src/app/components/graph/graph-viz.js

```diff
@@ -166,6 +166,19 @@ angular
                     },
                     show: true,
                 },
+                {
+                    id: 'docs-new-tab',
+                    content: 'View documentation in new tab',
+                    selector: 'node',
+                    tooltipText: 'Open the documentation for this node in a new browser tab',
+                    onClickFunction: function(event) {
+                        var target = event.target || event.cyTarget
+                        var unique_id = target.id();
+                        var url = $state.href('dbt.' + target.data('resource_type'), {unique_id: unique_id}, {absolute: true, inherit: false});
```

Comment:
I'm no javascript wizard, but I had to add the `inherit: false` parameter to get around the edge case where if a user first refocused on a node, and then within that refocused DAG they selected &rarr; "View documentation in new tab", it still functioned properly.

(without passing in this parameter, the new tab would revert back to the graph view if the DAG had been refocused via the menu option)